unsupported.txt

```diff
@@ -124,3 +124,8 @@ VENGEANCE | VENGEANCE [FEAT. JPEGMAFIA & ZILLAKAMI | JPEGMAF1A + Z1LLAKAM1] by D
 1000 Dreams by DJ Bobo
 Like This (The Sequel) by Technotronic
 Bhare Bazaar by Rishi Rich
+The Wanderer Between Life and Death by Arts Of Erebus 
```

Comment:
these could be discarded 👍

Reply:
Removed these and squashed commits into one. Have a look...